userspace/libsinsp/utils.h

```diff
@@ -100,18 +100,41 @@ class sinsp_utils
 ///////////////////////////////////////////////////////////////////////////////
 // little STL thing to sanitize strings
 ///////////////////////////////////////////////////////////////////////////////
+
+inline bool is_invalid_char(char c)
+{
+	if (c < -1)
```

Comment:
this doesn't respect the code formatting conventions. See https://github.com/draios/sysdig/blob/dev/coding_conventions.md.